src/data/endpoints/account.yaml

```diff
@@ -36,23 +34,39 @@ endpoints:
                   "email": "jsmith@mycompany.com",
                   "first_name": "John",
                   "last_name": "Smith",
-                  "network_helper": true,
                   "phone": "555-555-1212",
                   "state": "PA",
                   "zip": 19102,
+                  }
+                }' \
+                https://$api_root/$version/account
+  /account/settings:
+    group: Settings
+    type: resource
+    description: >
+      Manage default settings for your account.
+    methods:
+      GET:
+        response: account_settings
```

Comment:
Should probably include something like `is_managed`

Reply:
Done